tests/py/unit/test_build_atomic_layers.py

```diff
@@ -0,0 +1,53 @@
+from mat3ra.made.tools.analyze.lattice_planes import CrystalLatticePlanesMaterialAnalyzer
+from mat3ra.made.tools.analyze.other import get_local_extremum_atom_index
+from mat3ra.made.tools.build.slab.builders import AtomicLayersUniqueRepeatedBuilder, CrystalLatticePlanesBuilder
+from mat3ra.made.tools.build.slab.configuration import (
+    AtomicLayersUniqueRepeatedConfiguration,
+    CrystalLatticePlanesConfiguration,
+)
+from mat3ra.made.tools.build.slab.helpers import select_slab_termination
+from unit.fixtures.generated.fixtures import SrTiO3_BULK_MATERIAL
+
+NUMBER_OF_LAYERS = 1
+
+
+def process_termination(material, miller_indices, termination):
+    crystal_lattice_planes_config = CrystalLatticePlanesConfiguration(crystal=material, miller_indices=miller_indices)
+    crystal_lattice_planes_material = CrystalLatticePlanesBuilder().get_material(crystal_lattice_planes_config)
+
+    atomic_layers_config = AtomicLayersUniqueRepeatedConfiguration(
+        crystal=crystal_lattice_planes_material,
+        miller_indices=miller_indices,
+        termination_top=termination,
+        number_of_repetitions=NUMBER_OF_LAYERS,
+    )
+
+    atomic_layers_builder = AtomicLayersUniqueRepeatedBuilder()
+    atomic_layers_material = atomic_layers_builder.get_material(atomic_layers_config)
+
+    return atomic_layers_material
+
+
+def get_topmost_atom_element(slab):
+    center_coord = [0.5, 0.5, 0.5]
+    topmost_index = get_local_extremum_atom_index(slab, center_coord, "max", vicinity=1000.0)
+    return slab.basis.elements.get_element_value_by_index(topmost_index)
+
+
+def test_termination_translation_srtio3():
```

Comment:
Let's use parameterized logic - have a list of test cases with:

material:
termination_configs:
   - termination_formula
      topmost_element
   - termination_formula
      topmost_element